### PR TITLE
Minor develop/get-started-with-ton.mdx refactoring

### PR DESCRIPTION
**Pull request by arynyklas**
Minor develop/get-started-with-ton.mdx refactoring

## Why is it important?
To better understanding

### CODE REVIEW

**Review by SsNiPeR1 — CHANGES_REQUESTED** [review]
Line 159: grammar issue: you'll require access to THESE three essential tools.

File: docs/develop/get-started-with-ton.mdx [hunk 1]
```diff
@@ -152,11 +152,11 @@ If you choose the cloud development environment it's easy to get started by firs
 
 After completing this step, GitHub will create a special cloud workspace that allows you to access the VSCode Online IDE (Visual Code Online Integrated Development Environment).
 
-After this access is granted (it takes about 30 seconds for the codespace to start), and you'll have everything you need to get started, without ever needing to install Git, Node.js, or other developer tools.
+Once access is granted (the codespace typically starts in about 30 seconds), you'll have everything required to begin without the need to install Git, Node.js, or other developer tools.
 
 #### Local Development Environments
 
-To begin using a local development environment you’ll need access to 3 main tools. These include:
+To set up a local development environment, you'll require access to this three essential tools:
```
Comment (SsNiPeR1) [inline docs/develop/get-started-with-ton.mdx:159]: s/this/these

**Review by SsNiPeR1 — CHANGES_REQUESTED** [review]

**Review by SsNiPeR1 — APPROVED** [review]
All good now.

**Comment by SsNiPeR1** [comment]
@reveloper

**Comment by arynyklas** [comment]
@reveloper

**Comment by reveloper** [comment]
@arynyklas, @SsNiPeR1, Thank you!

**Pull request merged**